Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/UniqueIdentifierBasedRowIDProvider.cs

```diff
@@ -53,19 +53,19 @@ public BocListRow GetRowFromItemRowID (IReadOnlyList<IBusinessObject> values, st
       int rowIndex = tuple.Item1;
       string uniqueIdentifier = tuple.Item2;
 
-      if (rowIndex < values.Count && ((IBusinessObjectWithIdentity) values[rowIndex]).UniqueIdentifier == uniqueIdentifier)
+      if (rowIndex < values.Count && ((IBusinessObjectWithIdentity) values[rowIndex]!).UniqueIdentifier == uniqueIdentifier)
```

Comment:
Aren't those null-forgives obsolete, now? If so, could you please check in the PR for Value-indexer with null-forgiving operators?